index.html

```diff
@@ -225,7 +225,7 @@ <h4>Important caveat</h4>
     <div class="hiding">
       <h5>Generate Broadcast WAV</h5>
       <p><code>ffmpeg -i <em>input_file.wav</em> -c copy -write_bext 1 -metadata field_name='Content' <em>output_file.wav</em></code></p>
-      <p>This command will create a write a file in the Broadcast WAV (BWF) containing a BEXT chunk with related metadata.</p>
+      <p>This command will write a file in the Broadcast WAV format (BWF) containing a BEXT chunk with related metadata.</p>
```

Comment:
Is the official name not Broadcast Wave Format?

Reply:
The official name of `Broadcast Wave Format`, `BWF,` and `Broadcast WAV format` is `Broadcast Wave Format`, see the [EBU spec](https://tech.ebu.ch/docs/tech/tech3285.pdf). 

Happy to see BWF MetaEdit in here :)